discover/reference/articles/05-forms-field-types.markdown

```diff
@@ -53,22 +53,12 @@ is needed for multiple questions. For example, a product survey form might ask
 users to rate a list of their product's characteristics as Wonderful, Pretty
 Good, Not So Good, or Awful.
 
-![Figure x: A grid field.](../images/forms-grid.png)
+![Figure 8: A grid field.](../images/forms-grid.png)
 
 Numeric
 : Users enter numeric data (integers or decimals) into numeric fields.
 Non-number input is not accepted. For example, configure a numeric field that
 accepts integers to ask users how many children they have.
 
```

Comment:
Hi @rbohl,

The Grid field is fine for me. 😄

In regards to the Numeric field, I just would like to let you know that the Numeric field description mentions a subject (number of children) and the image related to this field refers to other subject (number of form entries). On the other hand, the Grid field refers to the same subject in both, field description and image.